terraform/modules/helm/postgresql/postgresql.yaml.tfpl

```diff
@@ -116,10 +116,10 @@ primary:
           published_date TIMESTAMP NOT NULL,
           UNIQUE (dataset_id)
         );
-      CREATE INDEX IF NOT EXISTS  dataset_source_config_dataset ON dataset_source_config(dataset_id);
+        CREATE INDEX IF NOT EXISTS  dataset_source_config_dataset ON dataset_source_config(dataset_id);
```

Comment:
@ManojCKrishna Why is the indentation required for the index statement?

Reply:
This line should be under obsrv database logic here - https://github.com/Sanketika-Obsrv/obsrv-automation/pull/40/files#diff-b58e82e9ecb675e0dd8424281c5497ce10e18be5ba74cb91ec7a78c4459efff8L39 
So it needs indentation.